src/main/java/core/basesyntax/model/Operations.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.model;
+
+public enum Operations {
+    R,P,S,B;
+
+    public static boolean contains(String value) {
+        try {
+            Operations.valueOf(value.toUpperCase());
+            return true;
+        } catch (IllegalArgumentException e) {
+            return false;
+        }
```

Comment:
let's refactor this method: catching RuntimeExceptions is not a good practice

try `public static Operations getByString(String value)` - return the enum from it or throw an exception